sources/Renderer/Metal/MTRenderContext.mm

```diff
@@ -62,12 +62,18 @@
     /* Initialize color and depth buffer */
     //MTLPixelFormat colorFmt = metalLayer_.pixelFormat;
 
+    // We have no need of the internal drawing timer
```

Comment:
That's an interesting change, too. Can you explain briefly what these two options do? Do we need to 'unpause' the view later?

Reply:
According to the Apple documentation MTKViews by default run an internal timer trying to redraw the view on a fixed interval. Setting 'paused' to true just signals to the view that we don't want that timer. This is primarily to configure the view based on Apple's guidelines and let it know we're drawing whenever we want to.
The other option disables the view's default management of the drawable sizes. This allows us to directly set the resolution to anything we want.

Reply:
Sounds good to me.